nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-restapi-processors/src/main/resources/docs/org.apache.nifi.processors.elasticsearch.PaginatedJsonQueryElasticsearch/additionalDetails.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<head>
+    <meta charset="utf-8" />
+    <title>PaginatedJsonQueryElasticsearch</title>
+    <link rel="stylesheet" href="../../../../../css/component-usage.css" type="text/css" />
+</head>
+<body>
+    <p>This processor is intended for use with the Elasticsearch JSON DSL and Elasticsearch 5.X and newer. It is designed
+    to be able to take a query from Kibana and execute it as-is against an Elasticsearch cluster in a paginated manner.
+    Like all processors in the "restapi" bundle, it uses the official Elastic client APIs, so it supports leader detection.</p>
+    <p>The query to execute can be provided either in the Query configuration property or in an attribute on a flowfile. In
+    the latter case, the name of the attribute (Expression Language is supported here) must be provided in the Query
+    property. If no Query property is provided and a flowfile is received, the content of the flowfile is used as the Query JSON.</p>
```

Comment:
This seems unclear to me based on the actual functionality.  How about:
```suggestion
    <p>The query JSON to execute can be provided either in the Query configuration property or in the content of the 
    flowfile.  If the Query Attribute property is configured, the executed query JSON will be placed in the attribute provided by this property.</p>
```

Reply:
Again a bit of documentation copied from the existing `JsonQueryElasticsearch` processor docs, but it's not very clear and your suggestion reads better (updated both sets of docs and corrected a grammatical error in `ConsumeElasticsearch` docs)